api/compose/api.go

```diff
@@ -56,6 +56,16 @@ type PortPublisher struct {
 	Protocol      string
 }
 
+// ContainerSummary hold high-level description of a container
+type ContainerSummary struct {
+	ID         string
+	Name       string
+	Project    string
+	Service    string
+	State      string
+	Publishers []PortPublisher
```

Comment:
Curious why this isn't `[]Port` as with the container

Reply:
same as port but includes an `URL`, as cloud integration has to provide external access domain name

Reply:
Couldn't we use the `HostIP` for that?

Reply:
I mean yeah sure, it's not an IP but a URL, ok ok ok